tools/backups2datalad/asyncer.py

```diff
@@ -111,13 +111,17 @@ async def asset_loop(self, aia: AsyncIterator[Optional[RemoteAsset]]) -> None:
                 if not downloading:
                     log.info("%s: Will download in a future run", asset.path)
                     self.tracker.mark_future(asset)
-                    if now - asset.created > timedelta(days=1):
-                        log.error(
-                            "%s: Asset created more than a day ago"
-                            " but SHA256 digest has not yet been computed",
-                            asset.path,
-                        )
-                        self.report.old_unhashed += 1
+                    if asset.asset_type == AssetType.BLOB:
+                        try:
+                            asset.get_raw_digest(DigestType.sha2_256)
+                        except NotFoundError:
+                            if now - asset.created > timedelta(days=1):
+                                log.error(
+                                    "%s: Asset created more than a day ago"
+                                    " but SHA256 digest has not yet been computed",
+                                    asset.path,
+                                )
+                                self.report.old_unhashed += 1
```

Comment:
wouldn't for https://github.com/dandi/dandisets/issues/126 we still need some kind of a crash here?  but then I am still not certain I understand how this fix is getting us the desired effect -- do we already have zarr's in that 000026?

Reply:
* There is no "crash" on Zarrs; as described [here](https://github.com/dandi/dandisets/issues/126#issuecomment-1043333568), encountering a Zarr just causes an error to be logged and the rest of the assets in the dataset to be ignored.

* This fix checks whether an asset being skipped (due to this or a previous asset being a Zarr or lacking a SHA256) lacks a SHA256, and only then does it log the given message and increment the "old assets missing SHA256" counter.  The check for blobness on line 114 is to avoid false positives from Zarrs, which don't have SHA256's.

Reply:
thanks, I think I got it. Let's proceed, thank you @jwodder